### PR TITLE
Bugfix: relay error of dependant field in radio options

### PR DESCRIPTION
**Pull request by nhaagen**
https://mantis.ilias.de/view.php?id=25037

### CODE REVIEW

**Comment by Amstutz** [comment]
@chfsx I added you here as reviewer, since your report has triggered this commit, I hope this is oke, thx for reporting.

**Review by Amstutz — APPROVED** [review]
@nhaagen , looks fine to me, thx for fixing.

File: src/UI/Implementation/Component/Input/Field/Radio.php [hunk 1]
```diff
@@ -132,19 +132,23 @@ public function withInput(PostData $post_input) {
...
 			foreach ($dep_fields as $name => $field) {
 				$filled = $field->withInput($post_input);
-
 				$content = $filled->getContent();
+
 				if ($content->isOk()) {
 					$values['group_values'][$name] = $content->value();
 				} else {
-					$error = true;
+					$clone = $clone->withError($name . $content->error());
```
Comment (klees) [inline src/UI/Implementation/Component/Input/Field/Radio.php]: We seem to be missing some delimiter between the name and the error (e.g. ": ") here, if I understand correctly.

**Review by klees — CHANGES_REQUESTED** [review]
LGTM, only some delimiter seems to be missing.

**Review by chfsx — APPROVED** [review]
Thanks a lot, will solve my problem!

File: src/UI/Implementation/Component/Input/Field/Radio.php (hunk 1 above)
Comment (nhaagen) [reply src/UI/Implementation/Component/Input/Field/Radio.php]: you are right; I added a "-".

**Review by klees — APPROVED** [review]

File: src/UI/Implementation/Component/Input/Field/Radio.php (hunk 1 above)
Comment (klees) [reply src/UI/Implementation/Component/Input/Field/Radio.php]: Give it some space " - " =) . Thx!

**Comment by klees** [comment]
Cherry-picked to trunk as well.

**Pull request merged**